crates/ibc/tests/mbt.rs

```diff
@@ -1,30 +1,30 @@
 extern crate alloc;
 
-mod runner;
+// mod runner;
```

Comment:
Would be nice to fix the build for MBT tests. Or maybe we should just remove them for now? (In a different PR.)

Reply:
Removed it entirely, as we were essentially maintaining dead code [a799452](https://github.com/cosmos/ibc-rs/pull/170/commits/a7994526f6a7c4c96834ca1a614c2c7993a544e0). 

We can bring it back properly with #148.